src/spdx_tools/spdx3/model/ai/ai_package.py

```diff
@@ -0,0 +1,87 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+from dataclasses import field
+from datetime import datetime
+from enum import Enum, auto
+from typing import Dict, List, Optional
+
+from spdx_tools.common.typing.type_checks import check_types_and_set_values
+from spdx_tools.spdx3.model import CreationInformation, ExternalIdentifier, ExternalReference, IntegrityMethod
+from spdx_tools.spdx3.model.software import SoftwarePurpose
+from spdx_tools.spdx.model import Package
+
+
+class SafetyRiskAssessmentType(Enum):
+    SERIOUS = auto()
+    HIGH = auto()
+    MEDIUM = auto()
+    LOW = auto()
+
+
+class AIPackage(Package):
+    energy_consumption: Optional[str] = None
+    standards_compliance: List[str] = field(default_factory=list)
+    limitations: Optional[str] = None
+    type_of_model: List[str] = field(default_factory=list)
+    information_about_training: Optional[str] = None
+    information_about_application: Optional[str] = None
+    hyperparameters: Dict[str, Optional[str]] = field(default_factory=dict)
+    data_preprocessing_steps: Optional[str] = None
+    model_explainability_mechanisms: Optional[str] = None
+    sensitive_personal_information: Optional[bool] = None
+    metrics_decision_thresholds: Dict[str, Optional[str]] = field(default_factory=dict)
+    metrics: Dict[str, Optional[str]] = field(default_factory=dict)
+    domain: Optional[str] = None
```

Comment:
This property doesn't have a `maxCount`, might be a problem in the spdx-3-model?

Reply:
Not sure, I will go with a list for now. Thanks for pointing this out!

Reply:
done